tests/unit/views/pacbio/PacbioSampleIndex.spec.js

```diff
@@ -1,6 +1,7 @@
 import Samples from '@/views/pacbio/PacbioSampleIndex'
 import { mount, localVue, store, Data, router } from '@support/testHelper'
 import Response from '@/api/Response'
+import { beforeEach, describe, expect, it } from 'vitest'
```

Comment:
Maybe these could be pulled out to the testHelper so they don't have to be imported for every test

Reply:
These are not necessary as these are imported by default. VSCode adds them not me. I gave up removing them because it was happening every time I added any of those identifiers and I haven't tried to fix it.